code/src/index.css

```diff
@@ -5,9 +5,198 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
 }
-
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* HEADER */
+.headermain {
+  display: flex;
+  font-family: Helvetica;
+  font-size: 14px;
+  font-weight: bold;
+  color: #ffffff;
+  border-bottom: solid 1px gray;
+}
+header {
+  display: flex;
+  justify-content: flex-start;
+  padding: 20px;
+  margin-left: 15px;
+  margin-bottom: 10px;
+}
+
+/* ALBUM-CARD WITH THE IMAGE AND THE ICONS */
+.album-card {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  padding: 5px 0;
+}
+.album-cover {
+  width: 300px;
+  height: 300px;
+}
+/* HOVER EFFECTS ON THE CARD */
+.container {
+  display: inline-block;
+  position: relative;
+}
+.container img {
+  display: block;
+}
+/* .container :hover {
+  filter: brightness(0.4);
+  opacity: 1;
+} */
+
+.album-card :hover .album-cover {
+  filter: brightness(0.4);
+  opacity: 1;
+}
+.album-card :hover .icons-wrapper {
+  display: flex;
+  width: 300px;
+}
+.album-card :hover {
+  text-decoration: underline;
+}
```

Comment:
Would it make more sense to have this styling on the `.artistname` class?

Reply:
Hey! unfortunately it does not work on artist-name class